products/storage/terraform.yaml

```diff
@@ -102,6 +102,12 @@ overrides: !ruby/object:Overrides::ResourceOverrides
         ignore_read: true
   Object: !ruby/object:Overrides::Terraform::ResourceOverride
     exclude: true
+  HmacKey: !ruby/object:Overrides::Terraform::ResourceOverride
```

Comment:
Mind adding an `exclude: true` in inspec.yaml + ansible.yaml for this resource? Our system gets tripped up sometimes with new resources and they cause generation to fail.